src/Microsoft.Health.Dicom.Api/Registration/DicomServerApplicationBuilderExtensions.cs

```diff
@@ -33,8 +34,10 @@ public static IApplicationBuilder UseDicomServer(this IApplicationBuilder app)
 
             app.UseHealthChecksExtension(new PathString(KnownRoutes.HealthCheck));
 
-            IOptions<FeatureConfiguration> featureConfiguration = app.ApplicationServices.GetRequiredService<IOptions<FeatureConfiguration>>();
+            // Update Fellow Oak DICOM services to use ASP.NET Core's service container
+            DicomSetupBuilder.UseServiceProvider(app.ApplicationServices);
```

Comment:
I am unsure if there is a better place to hook this in. We need to configure `fo-dicom` to use our service container.